Header header
uint16 joint_sensor
uint16 servo1_sensor
uint16 servo2_sensor
float64 rege_current
float64 servo_current
uint16 u1
uint16 u2
uint16 D1
uint16 D2
